src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';').map(
+    element => element.split(':')).filter(
+    property => property.length !== 1);
```

Comment:
Записуй методи ланцюжком у такому форматі
```suggestion
  const properties = sourceString.split(';')
    .map(element => element.split(':'))
    .filter(property => property.length !== 1);
```